src/maintainer/adding_pkgs.rst

```diff
@@ -638,18 +638,61 @@ go to the root repository directory and run the
 ``.scripts/run_docker_build.sh`` script.  This requires that you have docker
 installed on your machine.
 
-You need to define an environment variable named ``CONFIG``. Its value must be
-the name of one of the three YAML configuration files in the ``.ci_support``
-directory (either ``linux64``, ``osx64``, or ``win64``). As an example, you can
-invoke the command as follows.
+You need to define an environment variable named ``CONFIG``. Its value must correspond
+the name of one of the YAML configuration file in the ``.ci_support``
+directory (for all practical purposes, only ``linux64`` is used here). As an example, you can
+invoke the command as follows. 
 
 .. code-block:: sh
 
     $ cd staged-recipes
     $ CONFIG=linux64 ./.scripts/run_docker_build.sh
 
-Once built, you can find the finished package under ``staged-recipes/build_artifacts``.  
+Once built, you can find the finished package under ``staged-recipes/build_artifacts``. In order to run cuda-enabled builds locally, you need to set the ``IMAGE_NAME`` environment variable alongside the ``CONFIG`` environment variable (see the following for cuda 10.2 and 11.2). 
 
+.. code-block:: sh
+
+    $ cd staged-recipes
+    $ CONFIG=linux64 IMAGE_NAME=quay.io/condaforge/linux-anvil-cos7-cuda:10.2 ./.scripts/run_docker_build.sh
+
+.. code-block:: sh
+
+    $ cd staged-recipes
+    $ CONFIG=linux64 IMAGE_NAME=quay.io/condaforge/linux-anvil-cuda:11.2 ./.scripts/run_docker_build.sh
+
+If you are trying to run ``osx64`` builds, you can directly invoke ``run_osx_build.sh`` script. 
+
+.. code-block:: sh
+
+    $ cd staged-recipes
+    $ .scripts/run_osx_build.sh
+
+
+Or you can build using your own conda-build environment (``conda install conda-build`` or ``mamba install boa``) and the run the command (``conda build recipes/RECIPE_DIR`` or ``mamba build recipes/RECIPE_DIR``). However, if the recipe has additional variables like ``cuda_compiler_version`` in the case of cuda-enabled builds. Then, you would need to set that to ``"None"`` when running on osx.
+
+.. code-block:: sh
+
+    $ cd staged-recipes
+    $ cuda_compiler_version="None" mamba build recipes/RECIPE_DIR
+
+Running tests locally for feedstocks
+....................................
+
+Simply invoke the ``build-locally.py`` script in the root of the feedstock repository and select the appropriate build. 
+
+.. code-block:: sh
+
+    $ cd staged-recipes
+    $ python build-locally.py
+
+Or append the call with the name of the run from the ``.ci_support`` directory (without the ``.yaml`` extension).
+
+.. code-block:: sh
+
+    $ cd staged-recipes
+    $ python build-locally.py linux_64_c_compiler_version9cuda_compiler_versionNonecudnnundefinedcxx_compiler_version9numpy1.19python3.9.____cpython
+
+Note: for ``osx`` runs, you will need to set the ``export OSX_SDK_DIR`` when using ``python build-locally.py``, for example ``export OSX_SDK_DIR=/opt``.
```

Comment:
This is not necessary
```suggestion
```

Reply:
You mean it is not necessary to add it here or it is not necessary in general? My runs always fail if I don't set it... but maybe because I am doing something wrong!

Reply:
It's not necessary in general. It is set by `build-locally.py`

Reply:
Okay,  I have no idea why I always have problems with this one 👀

Reply:
Please open a separate issue for that.